### ISSUE TITLE
allennlp command prints extra logging message

### ISSUE THREAD

**Issue by schmmd** [allenai/allennlp#2005]
Starting in b5087e76d5127960e8ec64a1a7bd521f7f0c175b AllenNLP started printing `[INFO/MainProcess] process shutting down` on the last line.

```
$ allennlp --version                                                                                                                                                                                               allennlp 0.6.2-unreleased
[INFO/MainProcess] process shutting down
```

**Comment by schmmd** [comment]
@joelgrus not the world's most severe issue--but curious if you have any ideas on how to remove the extra logging.

**Comment by matt-gardner** [comment]
Yeah, I found that annoying too.  Looking at that PR, my best guess is it comes from this line: https://github.com/allenai/allennlp/blob/e4f41e7883a54bc6773617f6267b435f5c31413e/allennlp/data/iterators/multiprocess_iterator.py#L12

**Comment by joelgrus** [comment]
probably the fix is to only instantiate that logger if you actually need it, I can assign this to myself

**Comment by schmmd** [comment]
Fixed by #2166
